examples/examples-gpu/nyc-taxi/rf-scikit.ipynb

```diff
@@ -4,47 +4,29 @@
    "cell_type": "markdown",
```

Comment:
<p>is it worth having an expandable section for monitoring resources at the top of the notebook, like the way the rapids notebook does it? I'm cool with either way just a thought</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>yeah good idea! I'll add one and mention <code>htop</code></p>

Reply:
ok I added one in https://github.com/saturncloud/examples/pull/58/commits/94a42075ffafc352d59de8aaf17620c69a6a9194. I also included `nvidia-smi` in that, figured it's a good way to PROVE that scikit-learn isn't using the GPU